py/picca/delta_extraction/data_catalogues/desi_data.py

```diff
@@ -32,6 +32,30 @@
 defaults.update(defaults_quasar_catalogue)
 
 
+def merge_new_forest(forests_by_targetid, forests_by_pe):
+    """Merge forests_by_pe and forests_by_targetid as Forest instances.
+
+    Arguments
+    ---------
+    forests_by_targetid: dict
+    Dictionary were forests are stored. Its content is modified by this
+    function with the new forests.
+
+    forests_by_pe: str
+    Name of the file to read
```

Comment:
docstring error: forests_by_pe is a dictionary.

Reply:
Well spotted, addressed in new commit.